Update 03.player_movement_code.rst (#9672)
godotengine/godot-docs: getting_started/first_3d_game/03.player_movement_code.rst

|image9|

.. note::

>     If your preview image does not look like the screenshots above, try deleting the *Camera3D* node. 
>     And first set the Rotation X ``-45`` for *CameraPivot*, and then repeat the instructions after creating *Camera3D* node.

Reviewer: ```suggestion
    If your preview image does not look like the screenshots above, try deleting the *Camera3D* node. 
    Set the Rotation X on *CameraPivot* to ``-45`` degrees, then repeat the instructions after creating the *Camera3D* node.
```

>     Set the Rotation X on *CameraPivot* to ``-45`` degrees, then repeat the instructions after creating the *Camera3D* node.

Reviewer 2: ```suggestion
    Set the X-axis Rotation of *CameraPivot* to ``-45`` degrees, then repeat the instructions after creating the *Camera3D* node.
```
Or similar, "Rotation X" sounds incomplete
